shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -49,6 +59,9 @@ public TextInputPlugin(
       @NonNull PlatformViewsController platformViewsController) {
     mView = view;
     mImm = (InputMethodManager) view.getContext().getSystemService(Context.INPUT_METHOD_SERVICE);
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O)
+      afm = view.getContext().getSystemService(AutofillManager.class);
+    else afm = null;
```

Comment:
```suggestion
      afm = Build.VERSION.SDK_INT >= Build.VERSION_CODES.O ? view.getContext().getSystemService(AutofillManager.class) : null;
```

Reply:
SDK version check doesn't seem to work with ternary expressions, the linter started complaining when I applied the suggested change.